phase1-coordinator/src/coordinator_state.rs

```diff
@@ -2747,16 +2747,26 @@ impl CoordinatorState {
             .iter()
             .chain(self.current_verifiers.clone().iter())
             .filter_map(|(participant, participant_info)| {
-                if !self.is_coordinator_contributor(&participant)
-                    && participant_info
-                        .locked_chunks
-                        .values()
-                        .find(|lock| {
-                            let elapsed = now - lock.lock_time;
-                            elapsed > participant_lock_timeout
-                        })
-                        .is_some()
-                {
+                let exceeded_chunk_names: Vec<String> = participant_info
+                    .locked_chunks
+                    .values()
+                    .filter(|lock| {
+                        let elapsed = now - lock.lock_time;
+                        elapsed > participant_lock_timeout
+                    })
+                    .map(|lock| lock.chunk_id.to_string())
+                    .collect();
+
+                if !self.is_coordinator_contributor(&participant) && !exceeded_chunk_names.is_empty() {
+                    let exceeded_chunks_string: String = exceeded_chunk_names.join(", ");
+
+                    tracing::warn!(
+                        "Dropping participant {} because it has exceeded the maximum ({}) allowed time \
+                        it is allowed to hold a lock (on chunks {}).",
+                        participant,
+                        chrono_humanize::HumanTime::from(participant_lock_timeout),
+                        exceeded_chunks_string,
+                    );
```

Comment:
@kellpossible this part has the same logic, but better reporting on what chunks caused the drop, right?

Reply:
@ibaryshnikov yes that was my intention, to report exactly which chunk locks had their limit exceeded.